src/modules/mainPage/components/TiresFilter.tsx

```diff
@@ -0,0 +1,256 @@
+      color: BASE_COLORS.DEFAULT_BLUE,
+    },
+    "&.Mui-focused .MuiOutlinedInput-notchedOutline": {
+      borderColor: BASE_COLORS.DEFAULT_BLUE,
+    },
+    "&.Mui-focused .MuiInputLabel-root": {
+      color: BASE_COLORS.DEFAULT_BLUE,
+    },
+  },
+  "& .MuiAutocomplete-clearIndicator": {
+    color: FILTER_COLORS.BUTTON_RESET_FILTER,
+  },
+  //OPTION COLOR/FONTS
+  "& + .MuiAutocomplete-popper .MuiAutocomplete-option": {
+    fontFamily: FONTS.MAIN_TEXT_FAMILY,
+  },
+  "& + .MuiAutocomplete-popper .MuiAutocomplete-option:hover": {
+    backgroundColor: FILTER_COLORS.DEFAULT_BLUE_INACTIVE,
+  },
+  "& + .MuiAutocomplete-popper .MuiAutocomplete-option[aria-selected='true']:hover":
+    {
+      backgroundColor: `${FILTER_COLORS.DEFAULT_BLUE_INACTIVE} !important`,
+      color: `${FILTER_COLORS.TEXT_MAIN} !important`,
+    },
+  "& + .MuiAutocomplete-popper .MuiAutocomplete-option[aria-selected='true']": {
+    backgroundColor: `${BASE_COLORS.DEFAULT_BLUE} !important`,
+    color: `${BASE_COLORS.BACKGROUND_WHITE}`,
+  },
+});
+
+function TiresFilter() {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const history = useNavigate();
+  const filtersParams = useSelector(selectFilterData());
+  const [width, setWidthValue] = useState("");
+  const [profile, setProfileValue] = useState("");
+  const [diametr, setDiametrValue] = useState("");
+  const [season, setSeasonValue] = useState("");
+  const [brand, setBrandValue] = useState("");
+
+  const handleSearchButton = useCallback(() => {
+    dispatch(actions.setSelectedWidth(width));
+    dispatch(actions.setSelectedProfile(profile));
+    dispatch(actions.setSelectedDiametr(diametr));
+    dispatch(actions.setSeasonChange([season]));
+    dispatch(actions.setBrandChange([brand]));
+    history(
+      `shop/?price=${JSON.stringify([
+        Math.min.apply(null, filtersParams.prices),
+        Math.max.apply(null, filtersParams.prices),
+      ])}&width=${JSON.stringify(width)}&profile=${JSON.stringify(
+        profile
+      )}&diametr=${JSON.stringify(diametr)}&season=${JSON.stringify(
+        season === t("summer")
+          ? "summer"
+          : season === t("winter")
+          ? "winter"
+          : "all-season"
+      )}&brand=${JSON.stringify(brand)}`,
+      { replace: true }
+    );
+  }, [
+    brand,
+    diametr,
+    filtersParams.prices,
+    history,
+    profile,
+    season,
+    t,
+    width,
+    dispatch,
+  ]);
+
+  const handleAutocompleteChange = useCallback(
+    (type: FieldType) =>
+      (
+        event: SyntheticEvent,
+        value: unknown,
+        reason: AutocompleteChangeReason
+      ) => {
+        if (typeof value !== "string") return;
+        if (type === "width") setWidthValue(value);
+        if (type === "profile") setProfileValue(value);
+        if (type === "diametr") setDiametrValue(value);
+        if (type === "season") setSeasonValue(value);
+        if (type === "brand") setBrandValue(value);
```

Comment:
Use `switch/case` instead of `if` ladder

Reply:
done